tests/phpunit/Mutator/AbstractMutatorTestCase.php

```diff
@@ -145,7 +145,7 @@ protected function setUp(): void
         $mutants = [];
 
         foreach ($mutations as $mutation) {
-            $mutatorVisitor = new SimpleMutatorVisitor($mutation);
+            $mutatorVisitor = new MutatorVisitor($mutation);
```

Comment:
I ran into some issues with the `SimpleMutatorVisitor`.
Mainly that it calls 'canMutate' on every node again, which isn't something the 'normal' implementation does, so i've opted to replace it with the normal variant